packages/bottom-tabs/package.json

```diff
@@ -1,21 +1,43 @@
 {
   "name": "@navigation-ex/bottom-tabs",
   "version": "0.0.1",
-  "main": "src/index",
   "license": "MIT",
+  "main": "lib/commonjs/index.js",
+  "react-native": "src/index.tsx",
+  "module": "lib/module/index.js",
+  "types": "lib/typescript/src/index.d.ts",
+  "files": [
+    "src",
+    "lib"
+  ],
+  "scripts": {
+    "prepare": "bob build",
+    "clean": "del lib"
```

Comment:
Do you need local scripts at all? It's a common thing to run everything from the root dir